CRM/Upgrade/Incremental/php/FiveSeventySix.php

```diff
@@ -47,6 +47,7 @@ public function upgrade_5_76_alpha1($rev): void {
       $this->addTask('Migrate event cart ID', 'migrateEventCartID');
     }
     $this->addTask('Update civicrm_mailing to permit deleting records from civicrm_mailing_job', 'updateNewCiviMailFields');
+    $this->addTask('Remove Foreign Key References from cache tables', 'removeForiegnKeyReferencesCacheTables');
```

Comment:
@seamuslee001 Sorry to be pedantic.. can you correct the spelling to:
```suggestion
    $this->addTask('Remove Foreign Key References from cache tables', 'removeForeignKeyReferencesCacheTables');
```